src/controllers/userController.js

```diff
@@ -0,0 +1,25 @@
+const userService = require("../services/userService");
+const { catchAsync } = require("../utils/error");
+const { raiseCustomError } = require("../utils/error");
+
+const signIn = catchAsync(async (req, res) => {
+  const { code } = req.query;
+  if (!code) {
+    raiseCustomError("KEY_ERROR", 401);
+  }
+
+  const data = await userService.getKakaoAccessToken(code);
+  const kakaoAccessToken = data["access_token"];
+  const kakaoUserInfo = await userService.getKakaoUserInfo(kakaoAccessToken);
+  const kakaoId = kakaoUserInfo["id"];
+  const name = kakaoUserInfo.properties["nickname"];
+  const email = kakaoUserInfo["kakao_account"]["email"];
+
+  const userInfo = await userService.getUserByKaKaoId(kakaoId, name, email);
+
+  res.status(200).json(userInfo);
```

Comment:
```suggestion
  return res.status(200).json(userInfo);
```
실제 return keyword가 빠져도 이후 로직이 없기 때문에 동작에는 문제가 없지만 method의 실행문 안에서 response를 보내준 이후로 더 처리할 코드가 없다면 return을 통해 확실히 마무리 해주시는 것이 좋습니다